linode/manager
# change: [M3-8418] - Add documentation on changeset best practices

## Description 📝
Our team has some cultural knowledge about changeset formatting that isn't documented anywhere, but we apply it as part of the PR review process and when we review changelogs as part of the release process. This PR documents our best practices that so that contributors inside and outside our team have the same guidelines for quality changesets.

## Changes  🔄
- Updates the contributing guidelines (`CONTRIBUTING.md`) with the a new section "Writing a changeset", which lists best practices for a good changeset
   - Links to that subsection in the section "Submitting a pull request"
- Updates the checklist section of the PR template (`PULL_REQUEST_TEMPLATE.md`) to link to the best practices where 'changeset' is mentioned

## How to test 🧪

### Verification steps
(How to verify changes)
- Read through the changes and check for typos.
- Think about if there are any best practices that this PR missed and we should add, or anything we've been doing that we should change.

## As an Author I have considered 🤔

*Check all that apply*
